src/integrationTest/java/org/opensearch/security/plugin/TransportIndexDocumentIntoSystemIndexAction.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ */
+
+package org.opensearch.security.plugin;
```

Comment:
Nit: As the new classes in the `org.opensearch.security.plugin` package are only used by the  `SystemIndexTests` class, shouldn't these moved into a common package? Otherwise, their purpose might be unclear.

Reply:
Rearranged this to put the test in a package called `systemindex` and the sample plugins (and associated classes) in a subpackage named `sampleplugin`